src/database/utilities/createRecord.js

```diff
@@ -191,6 +191,8 @@ function createStocktakeBatch(database, stocktakeItem, itemBatch) {
     stocktake: stocktakeItem.stocktake,
     itemBatch: itemBatch,
     snapshotNumberOfPacks: numberOfPacks,
+    // Assume counted quantity === snapshot quantity, when StocktakeBatch is created
+    countedNumberOfPacks: numberOfPacks,
```

Comment:
Argh this assumption breaks my previous comment's line of thinking :/

Reply:
They really haven't "counted" anything at this point, so it's a bit of a misappropriation to automatically set it it `numberOfPacks`. That leaves you with no point of which to distinguish what has and hasn't been counted by the stocktaker.

Reply:
I suppose you would need to use the snapshot quantity for the totalQuantity of the stocktakeItem if the numberOfPacks for a batch was null, but that isn't particularly hard.

Reply:
Pre this PR the placeholder for actual quantity is 'No change', this implies there is no change to quantity for a given item, so for the user it does not suggest 'Not counted'.

I see your point thought.. how does a user distinguish between counted item and an item that is counted but the quantity has not changed..

Reply:
As per our conversation, will change `No Change` to `Not Counted` and keep exisiting (pre this PR) logic